### ISSUE TITLE
examples/glyphs/data_tables.ipynb error

### ISSUE THREAD

**Issue by Maggie-M** [bokeh/bokeh#2095]
In the last cell of the notebook, when this command is run:

```python
VBox(children=[plot, data_table]).html
```
This error is returned

```
Javascript error adding output!
ReferenceError: Bokeh is not defined
See your browser Javascript console for more details.
```

**Comment by damianavila** [comment]
FIx by #2101

**Comment by fpliger** [comment]
Confirmed it looks good but notebook has a new error because of recent interface changes (this new bug is being fixed by #2246). So I'm closing this issue as fixed as @damianavila mentioned.
